v-next/example-project/test/node/example-test.ts

```diff
@@ -22,4 +22,31 @@ describe("Example EDR based test", () => {
 
     assert.equal(blockNumberAfter, `0x5`);
   });
+
+  it("should show stack traces when a transaction reverts", async () => {
+    const connection = await hre.network.connect();
+
+    const Revert = await connection.ethers.getContractFactory("Revert");
+    const revert = await Revert.deploy([]);
+
+    try {
+      await revert.boom();
+    } catch (error) {
+      assert.ok(error instanceof Error, "Expected an error to be thrown");
+
+      assert.ok(
```

Comment:
no need to change it, but there's an [assert.match](https://nodejs.org/api/assert.html#assertmatchstring-regexp-message) that better captures this type of assertion.

Reply:
Thanks! I looked for something with "includes" semantics and couldn't find any 😅